modular_skyrat/master_files/code/modules/roundstart/lawsets.dm

```diff
@@ -0,0 +1,34 @@
+/datum/station_trait/unique_ai
+	report_message = "For experimental purposes, this station AI might show divergence from default lawset. To make your life miserable, we have removed your additional lawset boards. Good luck!"
+// Don't roll the default lawsets, please!
+/pick_weighted_lawset()
+	var/datum/ai_laws/lawtype
+	var/list/law_weights = CONFIG_GET(keyed_list/law_weight)
+	if(HAS_TRAIT(SSstation, STATION_TRAIT_UNIQUE_AI))
+		law_weights -= "asimov"
```

Comment:
take safeguard out of the weights if unique AI is rolled as well

Reply:
ok gamer it is done

Reply:
god bless